src/dfx/src/lib/project/templates.rs

```diff
@@ -226,7 +226,7 @@ pub fn builtin_templates() -> Vec<ProjectTemplate> {
             get_archive_fn: assets::new_project_js_files,
         },
         category: ProjectTemplateCategory::Support,
-        post_create: vec![],
+        post_create: vec![NPM_INSTALL.to_string()],
```

Comment:
This should remove the post create fields from the templates that require dfx_js_base, and add them here.

Reply:
I believe I have done this now